src/tacacs/nss/0001-Modify-user-map-profile.patch

```diff
@@ -564,9 +564,9 @@ index 79e62b9..ecfa0b0 100644
 +    useradd_info_t *user;
 +
 +    user = &useradd_grp_list[MIN_TACACS_USER_PRIV];
-+    user->gid = 999;
++    user->gid = 100;
```

Comment:
is 100 conflict with existing groups?

Reply:
It is a linux built-in group ('user') and I don't think it is used anywhere in sonic.